packages/esm-patient-printing-app/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@openmrs/esm-patient-printing-app",
+  "version": "10.1.0",
+  "license": "MPL-2.0",
+  "description": "Patient Label Printing for the OpenMRS SPA",
+  "browser": "dist/openmrs-esm-patient-printing-app.js",
+  "main": "src/index.ts",
+  "source": true,
+  "scripts": {
+    "start": "openmrs develop",
+    "serve": "webpack serve --mode=development",
+    "debug": "npm run serve",
+    "build": "webpack --mode production --color",
+    "analyze": "webpack --mode=production --env analyze=true",
+    "lint": "cross-env eslint src --ext tsx,ts --fix --max-warnings=0",
+    "test": "cross-env TZ=UTC jest --config jest.config.js --verbose false --passWithNoTests --color",
+    "test:watch": "cross-env TZ=UTC jest --watch --config jest.config.js --color",
+    "coverage": "yarn test --coverage",
+    "typescript": "tsc",
+    "extract-translations": "i18next 'src/**/*.component.tsx' 'src/**/*.modal.tsx' 'src/**/*.extension.tsx' 'src/**/*.workspace.tsx' 'src/**/*.hook.tsx' 'src/index.ts' --config ../../tools/i18next-parser.config.js"
+  },
+  "browserslist": [
+    "extends browserslist-config-openmrs"
+  ],
+  "keywords": [
+    "openmrs"
+  ],
+  "homepage": "https://github.com/openmrs/openmrs-esm-patient-chart#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openmrs/openmrs-esm-patient-chart.git"
+  },
+  "bugs": {
+    "url": "https://github.com/openmrs/openmrs-esm-patient-chart/issues"
+  },
+  "dependencies": {
+    "@carbon/react": "^1.76.0"
+  },
+  "peerDependencies": {
+    "@openmrs/esm-framework": "6.x",
+    "react": "18.x",
+    "react-i18next": "11.x",
+    "react-router-dom": "6.x"
+  },
+  "devDependencies": {
+    "webpack": "^5.94.0"
```

Comment:
Your webpack dependency string should match the one used in other modules, specifically `"webpack": "^5.99.9"`. And we shouldn't see webpack being added to the `yarn.lock` file. Multiple versions of webpack can lead to hard to diagnose behaviour.